### DISCUSSION TITLE
Intermittent Workflow Run failures with "Error: Unable to resolve action `org-name/internal-action-repo-name@tag`, repository not found"

### DISCUSSION THREAD

**Discussion by sayhiben** [community/community#30877] [Enterprise]
My team has configured an internal Action repository in our Enterprise Cloud account as "Accessible from repositories in the 'org-name' organization." This internal Action is used across many of our top-level and reusable Workflows in a high-traffic repository in the same Organization. 

We have encountered several Workflow Run failures associated with errors in resolving the internal Action:

```shell
Prepare workflow directory
Prepare all required actions
Getting action download info
Error: Unable to resolve action `org-name/internal-action-repo@tag`, repository not found
```

Retrying the failed job seems to be enough to get the build to pass.

My suspicion is something happening on GitHub's network. [Based on the limits described in the documentation](https://docs.github.com/en/rest/overview/resources-in-the-rest-api#requests-from-github-actions), I don't believe we're hitting any rate limits. 

What can we do to stabilize this?

**Comment by kimxogus** [comment]
Having the same error in enterprise too :(
